ocean_lib/web3_internal/web3_overrides/test/test_contract.py

```diff
@@ -4,15 +4,15 @@
 #
 import pytest
 from ocean_lib.models.test.conftest import *  # noqa
-from ocean_lib.web3_internal.currency import to_wei
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
 from ocean_lib.web3_internal.test.test_contract_base import MyFactory
 from ocean_lib.web3_internal.web3_overrides.contract import CustomContractFunction
 
 
-def test_main(web3, config, dtfactory_address):
```

Comment:
I think we should remove DTFactory class. Not quite sure in this ticket.

Reply:
To avoid any import errors for now I haven't deleted them, but there are TODOs in place for that.